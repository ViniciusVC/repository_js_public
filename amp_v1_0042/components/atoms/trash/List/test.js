import React from 'react';
import { shallow } from 'enzyme';
import Component from './index';

describe('List >', () => {
    it('deve renderizar conforme snapshot', () => {
        const wrapper = shallow(<Component>children</Component>);
        expect(wrapper.dive().getElement()).toMatchSnapshot();
    });
});
