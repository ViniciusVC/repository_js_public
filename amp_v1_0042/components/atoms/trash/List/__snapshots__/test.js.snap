// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Listview > deve renderizar conforme snapshot 1`] = `
<ul
  className="sc-bdVaJa jaybQV"
>
  children
</ul>
`;
